Sources/Logging/DiagnosticsLogger.swift

```diff
@@ -257,9 +257,7 @@ private extension DiagnosticsLogger {
                 autoreleasepool {
                     outputPipe.fileHandleForWriting.write(data)
 
-                    guard let string = String(data: data, encoding: .utf8) else {
-                        return assertionFailure("Invalid data is logged")
```

Comment:
I'd rather keep this assertion failure in place for debugging purposes. Note that assertions only fire during debugging, not for production builds.

Reply:
yes , and also crash when using simulators

Reply:
It does in the Simulator, which is great as it can help you fix logging issues